app/middleware.js

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line @next/next/no-server-import-in-page
+import { NextResponse } from "next/server";
+
+export async function middleware(request) {
+  const authToken = request.cookies.get("rc_authToken")
```

Comment:
Where did you get this, `rc_authToken`? IIRC, the Rocket Chat Google OAuth returns only the `rc_token` and `rc_uid`

Reply:
It's essentially rc_token, I'll refactor it asap! Thank you!

Reply:
Done, I've refactored this out and added the click to chat button as well in the new  menu bar component🎉!